static/ui/scss/govuk-template.scss

```diff
@@ -310,10 +310,7 @@ button:focus {
     line-height: 1;
     text-decoration: none;
     text-rendering: optimizeLegibility;
-    border-bottom: 1px solid transparent;
```

Comment:
We don't really want to be changing library files. Is there some other way of overriding this?